Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-strategies/ContestStrategies.tsx

```diff
@@ -35,33 +34,31 @@ const ContestStrategies = () => {
         }
     }, [ selectedId, contestStrategies, dispatch ]);
 
-    useEffect(() => {
-        if (selectedStrategy) {
```

Comment:
Like I mentioned in another comment, the issue was `not` finished. The styling was fixed and there are no `errors` related to selecting a value from the `dropdown`. The dropdown's search option is not `optional`, but it does not affect the normal use of the dropdown. I think that having `search` enabled always is not a bad idea, as it will always be useful. Like you said, it will be discussed further.